main.py

```diff
@@ -160,8 +161,10 @@ def main():
     logging.info(f"{len(payload)} records to upsert into postgrest.")
 
     if payload:
-        res = requests.post(PGREST_ENDPOINT, headers=headers, json=payload)
+        res = requests.post(f"{PGREST_ENDPOINT}/traffic_reports", headers=headers, json=payload)
```

Comment:
this was my dumb mistake, forgetting to add this here. so this accounts for incidents not making it since yesterday afternoon